tests/ValidateSbrpTests/ValidateSbrpTests.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ValidateSbrpTests;
+
+public class ValidateSbrpTests
+{
+    private string[] Packages { get; set; }
+
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+
+    private const string VerionPattern = @"(\.\d)+([\-\w])*";
+
+    private string RepoRoot { get; set; }
+
+    public ITestOutputHelper output { get; set; }
+
+    public ValidateSbrpTests(ITestOutputHelper _output)
```

Comment:
The `_` prefix for parameter names is opposite of one of the standard style conventions used in .NET.  internal and private fields typically use the prefix.  This is a potential source of confusion.  https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

Reply:
Changed in https://github.com/dotnet/source-build-reference-packages/commit/ce1a70dc071c6a5adce2b2d0ca6285031ee12258. Made into `Output` and applied similar changes to GenerateScriptTests.cs for consistency.